File: VB/dxSampleGrid/Person.vb
Imports System
Imports System.Linq

Namespace dxSampleGrid
    Public Class Person
        Public Sub New(ByVal i As Integer)
            FirstName = "FirstName" & i
            Age = i * 10
            HasCar = i Mod 2 = 0
            Children = i Mod 4
            BirthDate = (New Date(2010, 1, 1)).AddDays(i)
        End Sub

        Public Property Age() As Integer
        Public Property BirthDate() As Date
        Public Property Children() As Integer
        Public Property FirstName() As String
        Public Property HasCar() As Boolean
    End Class
End Namespace

File: CS/dxSampleGrid/Person.cs
using System;
using System.Linq;

namespace dxSampleGrid {
    public class Person {
        public Person(int i) {
            FirstName = "FirstName" + i;
            Age = i * 10;
            HasCar = i % 2 == 0;
            Children = i % 4;
            BirthDate = new DateTime(2010, 1, 1).AddDays(i);
        }

        public int Age { get; set; }
        public DateTime BirthDate { get; set; }
        public int Children { get; set; }
        public string FirstName { get; set; }
        public bool HasCar { get; set; }
    }
}